cyto_dl/nn/encoder_decoder.py

```diff
@@ -0,0 +1,39 @@
+import torch
+
+
+class EncoderDecoder(torch.nn.Module):
+    def __init__(
+        self,
+        encoder,
+        decoder,
+        encoder_ckpt=None,
+        decoder_ckpt=None,
+        train_encoder=True,
+        train_decoder=True,
+        latent_norm=None,
+    ):
+        super().__init__()
+
+        self.encoder = self._init_model(encoder, encoder_ckpt, train_encoder)
+        self.decoder = self._init_model(decoder, decoder_ckpt, train_decoder)
+
+        self.latent_norm = latent_norm if latent_norm is not None else torch.nn.Identity()
+
+    def _init_model(self, model, ckpt, train):
+        if ckpt is not None:
+            state_dict = torch.load(ckpt)["state_dict"]
+            state_dict = {
+                k.replace("backbone.", ""): v
```

Comment:
can you add a comment about why this is necessary

Reply:
I'm thinking this guy isn't ready for prime time yet